### PR TITLE
Generated biomass plot

### PR DESCRIPTION
**Pull request by sbreitbart-NOAA**

### CODE REVIEW

File: R/plot_biomass.R [hunk 1]
```diff
@@ -0,0 +1,121 @@
+#' Plot Total Biomass
+#'
+#' @template dat
+#' @template model
```
Comment (kellijohnson-NOAA) [inline R/plot_biomass.R:4]: The use of `#' @template` is outdated. Guidelines suggest that package developers use `#' @inheritParams`. See guidance here on "[Reusing documentation](https://cran.r-project.org/web/packages/roxygen2/vignettes/reuse.html)".

File: R/plot_biomass.R [hunk 2]
```diff
@@ -0,0 +1,121 @@
+#' Plot Total Biomass
+#'
+#' @template dat
```
Comment (Schiano-NOAA) [inline R/plot_biomass.R:3]:
```suggestion
#' @inheritparams plot_recruitment
```

File: R/plot_biomass.R (hunk 1 above)
Comment (Schiano-NOAA) [inline R/plot_biomass.R:4]:
```suggestion

```

Comment (Schiano-NOAA) [reply R/plot_biomass.R]: Thanks for letting us know kelli! I have made the change following the guidance you have provided :)

**Pull request merged**